release.sh

```diff
@@ -28,17 +28,10 @@ if [[ -z "${JETBRAINS_HUB_TOKEN:-}" ]]
     exit 1
 fi
 
-NEW_VERSION=$1
-
-if [[ "$(uname)" == "Darwin" ]]
+if [[ "$(git branch --show-current)" != "master" ]]
   then
-    SED_ARG=(-i "")
-else
-    SED_ARG=(-i)
+    echo "Please use master branch"
```

Comment:
Should we move this to a variable to avoid DRY here? I mean, the name of the main/master branch?

Reply:
Yep, makes sense. https://github.com/zalando/intellij-swagger/pull/355/commits/7a92623525c2e97bb957e563233d235863f359b5